pkg/types/baremetal/platform.go

```diff
@@ -0,0 +1,59 @@
+package baremetal
+
+type BMC struct {
```

Comment:
what's BMC

Reply:
bare metal controller, a device in a server that allows you to manage it out of band

(obviously a comment would help here)

Reply:
Comment is coming https://github.com/openshift-metal3/kni-installer/pull/113